insta_user_names-filter/main/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	filter "github.com/codeuniversity/smag-mvp/insta_user_names-filter"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+)
+
+func main() {
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
+	groupID := utils.MustGetStringFromEnv("KAFKA_GROUPID")
+	changesTopic := utils.GetStringFromEnvWithDefault("CHANGES_TOPIC", "postgres.public.users")
+	namesTopic := utils.GetStringFromEnvWithDefault("NAMES_TOPIC", "user_names")
+
+	f := filter.New(kafkaAddress, groupID, changesTopic, namesTopic)
+
+	service.CloseOnSignal(f)
+	waitUntilClosed := f.Start()
```

Comment:
How can you call the `Start()` method on the type `UserNameFilter`?

Reply:
The UserNameFilter has/had the changestream.Filter embedded, which in turn has the worker.Worker embedded, which provided the Start method

Reply:
So is this some kind of inheritance?